torchaudio-contrib/io.py

```diff
@@ -0,0 +1,48 @@
+import torch
+import numpy as np
+import warnings
+from scipy.io import wavfile
+import librosa
```

Comment:
This means users need to install librosa even if they only read wav files. The import should only be done when needed.

Reply:
- The librosa import is to handle WAV file types which scipy cannot -- e.g. 24 bit files.

Reply:
Okay, let me refine: This means users need to isntall librosa even if they only read 16-bit wav files. The import should only be done when needed.